assets/queries/cloudFormation/elastic_search_domain_without_encryption_at_rest_options/query.rego

```diff
@@ -0,0 +1,40 @@
+package Cx
+
+CxPolicy [ result ] {
+  document := input.document
+  resource = document[i].Resources[name]
+  resource.Type == "AWS::Elasticsearch::Domain"
+  properties := resource.Properties
+  
+  exists_encryptionAtRestOptions := object.get(properties, "EncryptionAtRestOptions", "undefined") != "undefined"
+  not exists_encryptionAtRestOptions
+  
+      result := {
+                "documentId": 		input.document[i].id,
+                "searchKey":        sprintf("Resources.%s.Properties.EncryptionAtRestOptions", [name]),
+                "issueType":		   "MissingAttribute",
+                "keyExpectedValue": sprintf("Resources.%s.Properties.EncryptionAtRestOptions is defined", [name]),
+                "keyActualValue": 	sprintf("Resources.%s.Properties.EncryptionAtRestOptions is undefined", [name])
+              }
+}
+
+CxPolicy [ result ] {
+  document := input.document
+  resource = document[i].Resources[name]
+  resource.Type == "AWS::Elasticsearch::Domain"
+  properties := resource.Properties
+  
+  checks_encryption_at_rest_options(properties.EncryptionAtRestOptions)
```

Comment:
This function is only being used once, so it's unnecessary, this should be enough:

`properties.EncryptionAtRestOptions.Enabled  != true`